leetcode/src/540.c

```diff
@@ -0,0 +1,32 @@
+/**
+ * Time complexity: O(log n).
+ * Space complexity: O(1).
+ * description: The array has a pattern that consists in that the existing sub-array to 
```

Comment:
We can use Doxygen format here. 🙂
```suggestion
 * @details The array has a pattern that consists in of the existing sub-array to 
```

Reply:
Thanks for the advise. I made it in the last commit 😄